### PR TITLE
Fix security-scan pod log errors.

### PR DESCRIPTION
**Pull request by doflamingo721**
Linked issue:
https://github.com/rancher/cis-operator/issues/149

### CODE REVIEW

File: package/run.sh [hunk 1]
```diff
@@ -28,7 +28,7 @@ handle_error() {
       sleep infinity
   fi
   # Annotate self (pod) to signal "error"
-  if [[ -f "${ERROR_LOG_FILE}" ]]; then
+  if [[ -s "${ERROR_LOG_FILE}" ]]; then
```
Comment (prachidamle) [inline package/run.sh:31]: Can you explain why this change is needed?

Comment (doflamingo721) [reply package/run.sh:31]: `-f` checks if the file is a regular file or not and returns true if the file is a regular file (not a directory). So even if the error log file is empty, it will still return true because the file exists. Where as `-s` checks if file is not zero size. So if the error log file is empty, it won't execute the block.

**Review by prachidamle — APPROVED** [review]
Thanks for debugging this one, LGTM

**Review by dhruvmewada15 — APPROVED** [review]
LGTM
